src/udev/udev-builtin-net_id.c

```diff
@@ -205,10 +226,17 @@ static int dev_pci_onboard(sd_device *dev, const LinkInfo *info, NetNames *names
                 log_device_debug(dev, "dev_port=%lu", dev_port);
         }
 
+        /* Parse VF number if the interface is VF representor */
+        if (naming_scheme_has(NAMING_SR_IOV_R) && is_vf_repr(info))
+                parse_vf_repr_num(info, &repr_num);
```

Comment:
Please don't do the parsing twice.
```C
if (naming_scheme_has(NAMING_SR_IOV_R))
    (void) parse_vf_repr_num(info, &repr_num);
```

Reply:
Is it possible for the representor to be UINT_MAX?

Reply:
No, it is not... There is no existing HW that would support UINT_MAX number of VFs :-)